Content.Server/Medical/Components/CryoPodComponent.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Content.Server.Atmos;
+using Content.Shared.Atmos;
+using Content.Shared.Medical.Cryogenics;
+
+namespace Content.Server.Medical.Components;
+
+[RegisterComponent]
+[ComponentReference(typeof(SharedCryoPodComponent))]
```

Comment:
Shouldn't use compreferences unless absolutely necessary (they're not in this instance).

Reply:
Are you sure they're not necessary? When removing them, I get this error: "Component is not a valid reference type: SharedCryoPodComponent". I think it makes sense since the shared component is abstract yet the shared system code registers listeners on it, so the reference to the shared component has to exist somehow, no?

Reply:
You just need to update subscribers to use client / server versions but using the shared in the method signature is still fine.